tools/openapi-plugin/src/generators/client/generator.ts

```diff
@@ -1,18 +1,20 @@
 import {
     Tree,
     addProjectConfiguration,
+    formatFiles,
     generateFiles,
     joinPathFragments,
 } from '@nx/devkit';
 import openapiTS, { astToString } from 'openapi-typescript';
 import { loadConfig } from '@redocly/openapi-core';
 import { ClientGeneratorSchema } from './schema';
+import { addTsConfigPath } from '@nx/js';
```

Comment:
I can see that this function use `@nx/devkit` api internally. Is it better to call it or create a copy for our usage so we don't have to add `@nx/js` plugin into our dependencies?

Ref: [addTsConfigPath](https://github.com/nrwl/nx/blob/19e765f89a83f7bf0299116d4b424fc707161787/packages/js/src/utils/typescript/ts-config.ts#L61C17-L61C32)

Reply:
I think that's a good idea, yes.